IceCubesApp/Resources/Localization/Localizable.xcstrings

```diff
@@ -26160,6 +26160,45 @@
         }
       }
     },
+    "enum.translation-type.apple" : {
```

Comment:
FYI, when adding a new translation, I'm OK with using English localization as the key. It makes the process much simpler as we don't have to add the default english localization to every languages.

Reply:
I can revert the one commit that changes that for Apple Translate and DeepL